horreum-web/src/domain/reports/Reports.tsx

```diff
@@ -178,7 +178,7 @@ export default function Reports(props: ReportGroup) {
                                             _ => null /* errors because the config does not exist => OK */
                                         )
                                     }}
-                                    onImport={config => reportApi.importTableReportConfig(config)}
+                                    onImport={config => reportApi.importTableReportConfig(config as TableReportConfig)}
```

Comment:
what is the `config` type? we shouldn't be casting to `TableReportConfig`. 

You have fixed the API, we should us the correct type in the .tsx file

Reply:
theoretically we shouldn't cast but in this case `config` could be of three different types https://github.com/Hyperfoil/Horreum/pull/1610/files#diff-a3a05d2417739404a8de949561b4622db8476c09384cd5d9cf5dd745ddf169c4R11 because the `ImportButton` seems a shared component that is used in other places (in Schema and Test exports), thus for the sake of simplicity I just added the `TableReportConfig`.

IMO the cleanest approach would be to make the `ImportButton` generic such that we don't have a fixed set of types for the `onImport` function.